src/main/java/edu/usf/cutr/trackerlib/location/DouglasPeuckerDecimation.java

```diff
@@ -1,6 +1,5 @@
 /*
- * Copyright 2008 Google Inc., 2015 Cagri Cetin (cagricetin@mail.usf.edu),
- * University of South Florida
+ * Copyright 2008 Google Inc., 2015 University of South Florida (cagricetin@mail.usf.edu)
```

Comment:
@cagryInside Could you change this to `Copyright 2008-2015 Google Inc., University of South Florida (cagricetin@mail.usf.edu)`?

This is the more common format I've seen with date ranges.